index.html

```diff
@@ -0,0 +1,299 @@
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="self-container">
+          <p>Let's Connect</p>
+          <div class="list-nav">
+            <ul class="list-items skill-items">
+                <li>
+                    <a href="">
+                      <img src="images/Twitter.svg" alt="my social handle images" />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="">
+                      <img src="images/Vector.svg" alt="my social handle images"/><img src="images/Vector (1).svg" alt="my social handle images" />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="">
+                      <img src="images/Medium.svg" alt="my social handle images" />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="">
+                      <img src="images/Vector1.svg" alt="my social handle images" />
+                    </a>
+                  </li>
+                  <li>
+                    <a href=""><img src="images/Angeist.svg" alt="my social handle images" />
+                    </a>
+                  </li>
+            </ul>
+          </div>
+        </div>
+        <button type="button">Get my Resume</button>
+      </div>
+      <div class="language flex column">
+        <ul>
+          <li>
+            <div class="lang flex column">
+              <p class="lang-text flex">
+                <span class="txt">Language</span><span class="lang-line">></span>
+              </p>
+              <ul class="lang-div flex column">
+                <li class="lang-list flex">
+                  <img src="images/Js.svg" alt="js logo" />
+                  <p class="skill-text">JavaScript</p>
+                </li>
+                <li class="lang-list flex">
+                  <img src="images/Html.png" alt="html logo" />
+                  <p class="skill-text">HTML</p>
+                </li>
+                <li class="lang-list flex">
+                  <img src="images/css.png" alt="css logo" />
+                  <p class="skill-text">CSS</p>
+                </li>
+              </ul>
+            </div>
+          </li>
+          <li>
+            <div class="lang">
+              <p class="lang-text flex">
+                <span class="txt">Framework</span><span class="line">></span>
+              </p>
+            </div>
+          </li>
+          <div class="frame-line">
+            <hr/>
+          </div>
+          <li>
+            <div class="lang">
+              <p class="lang-text flex">
+                <span class="txt">Skills</span><span class="line">></span>
+              </p>
+            </div>
+          </li>
+          <div class="frame-line">
+            <hr />
+          </div>
+        </ul>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Please remove this section for the time being because it is unrelated to this milestone. You can add later, but for now, let's take it slowly.👌😊